cmd/admin-user-svcacct-add.go

```diff
@@ -50,8 +50,12 @@ var adminUserSvcAcctAddFlags = []cli.Flag{
 		Usage: "path to a JSON policy file",
 	},
 	cli.StringFlag{
-		Name:  "comment",
-		Usage: "personal note for the service account",
+		Name:  "name",
+		Usage: "friendly name for the service account",
+	},
+	cli.StringFlag{
+		Name:  "description",
```

Comment:
We can retain `comment` as a hidden alias? @donatello

Reply:
Add comment as extra and hidden option - I dont think it's possible to make it a "hidden alias" in the cli lib.